### DISCUSSION TITLE
I tried to install radare2 on 32bit windows xp but i got strange problem

### DISCUSSION THREAD

**Discussion by AirShoot125** [radareorg/radare2#21362] [Q&A]
I used next commands in windows cmd

git clone https://github.com/radare/radare2
cd radare2
cd sys
install.sh

After install.sh for a second, some window appeared, as if it was trying to do something, but could not, and immediately closed

r2 -w libname .so was next but instead of this i see that "r2 is not recognized as an internal or external command, operable program or batch"

How to solve it?

**Comment by trufae** [comment]
Shellscripts cant be executed on windows. The instructions to build radare2 on windows are:

* open cmd
* run preconfigure
* Run configure
* Run make
* Copy the prefix directory to c:\radare2 or wherever you want or set the path
